scripts/kind-delete.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -o errexit
+
+# desired cluster name; default is "kind"
+KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
+
+kind delete cluster --name ${KIND_CLUSTER_NAME}
```

Comment:
In other projects, I've seen this kind of thing as a make target, would that make sense here?

Reply:
Let me remove this file. I had added it during an attempt to add this to a CI test.